packages/manager/apps/dedicated/client/app/components/dedicated-cloud/service-pack/upgrade/steps/selection/selection.html

```diff
@@ -1,6 +1,6 @@
-<ovh-manager-no-default-means-of-payment
+<duc-ovh-manager-no-default-means-of-payment
```

Comment:
Maybe this should be moved to `ng-ovh-payment-method` as it looks related to payment method (as a next step)